README.md

```diff
@@ -61,6 +61,25 @@ $ grunt
 
 More information at [Running grunt](https://docs.moodle.org/dev/Grunt#Running_grunt) on Moodle's documentation site.
 
+### Hot reload with Shifter
+
+Shifter is a tool that takes the source JavaScript file `yui/src/button/button.js` and rebuilt the plugin's code each time you make changes to the source file.
+
+To install shifter globally with NPM:
+
+```sh
+npm install shifter@0.4.6 -g
+```
+
+During development you can run shifter to detect changes to your files:
```

Comment:
Maybe we should specify the place where the shift --watch command has to be executed